bin/init.sh

```diff
@@ -17,7 +17,7 @@ slug=${git_slug%.*}
 
 read -p "Repo slug: (leave blank for \"$slug\") " new_slug
 
-cd $(realpath $(dirname "$0")/..)
+pushd $(dirname "$0")/..) > /dev/null
```

Comment:
I get this error when running it:
```
$ ./bin/init.sh
Port number for new app: 7070
Replace `demo` package name with: test
Repo slug: (leave blank for "spring-boot-template") awesome
./bin/init.sh: line 20: syntax error near unexpected token `)'
./bin/init.sh: line 20: `pushd $(dirname "$0")/..) > /dev/null'
```
tried on bash and sh (it runs sh because of the shebang)
```
$ bash --version
GNU bash, version 4.4.23(1)-release (x86_64-apple-darwin17.5.0)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
 timja at potato in ~/projects/moj-reform/reform-common/spring-boot-template on remove-realpath-from-init*
$ sh --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Reply:
If its performant, might be better that this is run in docker by volume mounting, too many portability issues with the script.

Most people here do run macs, and just install realpath from homebrew

Reply:
Oops, typo. Should have been `pushd $(dirname "$0")/.. > /dev/null` without the trailing ) after the dots. Installing realpath would work, as would including a tiny C program that calls the realpath function that could be compiled and run by the script. I'm just not sure what the point of having it there is. Also if most people run macs, that's another reason to make it work ootb for them, no?

Reply:
its fine, works after your fix on my machine